.circleci/config.yml

```diff
@@ -89,7 +89,8 @@ jobs:
               helm dependency update
               cd -
               kubectl create namespace test
-              # debugging...
+              # Install the dask-gateway CRDs. It's not clear why helm doesn't
+              # do this for us.
```

Comment:
I think this is a helm3 feature, and that dask-gateway isn't packaged as a helm3 chart where the crds folder matters. Helm2 relied on a "helm hook" rather than a crd folder.

I think for now, until helm2 is dropped entirely by all users installing the dask-gateway helm chart, this is the best option.

/cc: @jcrist this relates to Chart.yaml's apiVersion field I think, and if you bump it to version 2 then it becomes helm2 incompatible.

reference: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

Reply:
So as a top-level chart, if the user uses `helm3` it still does the right thing. I didn't want to force all users to use helm3 by bumping up the `apiVersion`, but maybe we should?